ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleInitializer.java

```diff
@@ -109,6 +110,17 @@ public void onAutoCacheAdAvailable(String placementId) {
     // Unused
   }
 
+  public void updateCoppaStatus(int configuration){
+    switch (configuration) {
+      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE :
+        Vungle.updateUserCoppaStatus(true);
+        break;
+      case RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE :
+        Vungle.updateUserCoppaStatus(false);
+        break;
```

Comment:
How about the default value `TAG_FOR_CHILD_DIRECTED_TREATMENT_UNSPECIFIED`?

Reply:
@HuFangshuai For that value, there's no specified behavior which mirrors our unset state. "TAG_FOR_CHILD_DIRECTED_TREATMENT_UNSPECIFIED  if you don't want to indicate how you would like your content treated with respect to COPPA in ad requests." In this case we won't send a COPPA value in our requests.

There is an issue though. If a developer sets true/false, there's no way for them to set TAG_FOR_CHILD_DIRECTED_TREATMENT_UNSPECIFIED afterwards.